README.md

```diff
@@ -10,7 +10,7 @@ The uORB message definitions, which represent the counter-part of the ROS2 messa
 
 ## How are these messsage definitions generated?
 
-The PX4 Firmware Jenkins-based CI/CD deploys new messages or modifications to the current definitions with each build of the master branch of the PX4 Firmware. A future improvement will also include tagging and releasing this repository according to the tags/releases of the PX4 Firmware, for a more tide coupling between the message definitions.
+When uORB message definitions in the [PX4 firmware](https://github.com/PX4/Firmware) repository changes, a Jenkins-based CI/CD automatically generates and pushes updated ROS message definitions to this repository. The ROS message definitions are generated by the [`uorb_to_ros_msgs.py`](https://github.com/PX4/Firmware/blob/master/msg/tools/uorb_to_ros_msgs.py) script. One can also use this script to generate its own ROS message definitions for new or modified uORB messages.
```

Comment:
```suggestion
When uORB message definitions in the [PX4 firmware](https://github.com/PX4/Firmware) repository change, a Jenkins-based CI/CD automatically generates and pushes updated ROS message definitions to this repository. The ROS message definitions are generated by the [`uorb_to_ros_msgs.py`](https://github.com/PX4/Firmware/blob/master/msg/tools/uorb_to_ros_msgs.py) script. One can also use this script to generate its own ROS message definitions for new or modified uORB messages.
```

Reply:
It's not in Jenkins anymore, but rather in Github Actions.

Reply:
Aha! I found it still in the Jenkinsfile, but maybe that's just kept around but not running. And thanks for spotting the typo.